plugins/by-name/git-worktree/deprecations.nix

```diff
@@ -0,0 +1,9 @@
+{
+  optionsRenamedToSettings = [
+    "changeDirectoryCommand"
+    "updateOnChange"
+    "updateOnChangeCommand"
+    "clearJumpsOnChange"
+    "autopush"
```

Comment:
These will be renamed to snake case, but you've used kebab-case in the option declarations. I assume one is incorrect?

Reply:
lol.. thanks. Not sure how I messed that up.